std/exception.d

```diff
@@ -1485,11 +1485,13 @@ class ErrnoException : Exception
 {
     final @property uint errno() { return _errno; } /// Operating system error code.
     private uint _errno;
+    /// Constructor which takes an error message. The current global $(D errno) value is used as error code.
```

Comment:
Currently DDox turns `errno` here into a link to ErrnoException's errno property. Maybe this could use an explicit link to the global to avoid confusion.